### ISSUE TITLE
List or document all actions [Help / Advice]

### ISSUE THREAD

**Issue by mumblez** [rundeck/rundeck-cli#142]
can't seem to figure out what all the possible actions are for scm perform commands.

I'm particularly trying to work out how to get the projects in a CLEAN syncState from DELETE_NEEDED as it's showing 'Source file was deleted' on the ui.

Any help is much appreciated
Original request - https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/rundeck-discuss/A8uNmHBcofQ/kolEPRblAAAJ

Many Thanks

**Comment by gschueler** [comment]
Hi, this is a case where unfortunately, the API isn't sufficient to resolve the state :(.  Basically, as a tracked file was removed from git, Rundeck identifies that a job needs to be deleted to synchronize the state.  However, it does not automatically delete the job.  Additionally, the API does not present enough info for the client to know how to fix it :(.  I filed this enhancement request: https://github.com/rundeck/rundeck/issues/2330 because we need to add further information to the API response to indicate which job(s) would need to be deleted.

To resolve the state to CLEAN you will have to identify which jobs correspond to the deleted scm files, and then delete them
